### PR TITLE
Tweaks to support sign-up changes.

### PR DESCRIPTION
**Pull request by methnen**
Issues:
https://github.com/GigaOM/legacy-pro/issues/4799
https://github.com/GigaOM/legacy-pro/issues/4798

Related:
https://github.com/GigaOM/gigaom/pull/6487

### CODE REVIEW

File: components/templates/signup-form.php [hunk 1]
```diff
@@ -44,9 +44,22 @@
 	{
 		echo '<header>' . esc_html( $template_variables['header'] ) . "</header>\n";
 	}
+
+	if ( ! $is_advisory_signup && ! $is_individual_signup )
```
Comment (borkweb) [inline components/templates/signup-form.php]: Empty if statement

File: components/templates/signup-form.php [hunk 2]
```diff
@@ -44,9 +44,22 @@
...
+	{
+
```
Comment (wluo) [inline components/templates/signup-form.php]: empty clause.

File: components/templates/signup-form.php [hunk 3]
```diff
@@ -44,9 +44,22 @@
...
+	} // END if
 	?>
 	<div id="form-wrapper">
 		<form id="go-subscriptions-signup" class="boxed" method="post" action="<?php echo esc_url( $template_variables['ajax_url'] ); ?>">
+			<?php
+			if ( ! $is_advisory_signup && ! $is_individual_signup )
```
Comment (wluo) [inline components/templates/signup-form.php]: i think this is where the above empty `if` was supposed to be?

**Comment by borkweb** [comment]
:+1:

**Pull request merged**